### DISCUSSION TITLE
Customize markdown wysiwyg editor

### DISCUSSION THREAD

**Discussion by py6jlb** [OrchardCMS/OrchardCore#11845] [Q&A] [answered]
Hello. I'm trying to customize markdownwysiwyg([doc](https://docs.orchardcore.net/en/dev/docs/reference/modules/Markdown/#custom-editors)), but it doesn't work for me. I added an editor view to the theme module, but the editor didn't change. From the documentation, I did not understand how to add a named form. Any ideas how to fix this?

**Comment by ns8482e** [comment, marked as answer]
@py6jlb you can only add editor either in host project, module or  in admin theme but not in front end theme

**Reply by py6jlb** [reply]
Thank you. I moved the editor to a module and everything worked.
